frontend/src/app/components/crm/leads/leads.component.html

```diff
@@ -1,4 +1,9 @@
 <div class="container">
+  <br/>
+  <br/>
```

Comment:
Small thing here. I wouldn't use break tags here to create spacing. Instead, I'd resort to setting the margin-top property on this button.

Reply:
will fix